vendor/github.com/konveyor/openshift-velero-plugin/LICENSE

```diff
@@ -0,0 +1,201 @@
+Apache License
```

Comment:
I don't think these files should be showing up under vendor. It looks like this repo is including itself as its own dependency here, putting files under vendor/ that also exist at the top level.

Reply:
@sseago, you're right, that was a mistake, it's fixed now, thanks

Reply:
@sseago as you found out there is a typo in the project path. Rebuilding it.